src/db/schemas/settlement.model.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  pgEnum,
+  pgTable,
+  real,
+  timestamp,
+  varchar,
+} from "drizzle-orm/pg-core";
+import { createInsertSchema, createSelectSchema } from "drizzle-zod";
+import type { z } from "zod";
+
+import { splitTypeArr } from "@/common/enums";
+
+import groupModel from "./group.model";
+import userModel from "./user.model";
+
+export const splitTypeEnum = pgEnum("splitType", splitTypeArr);
+
+const settlementModel = pgTable("settlement", {
+  id: varchar({ length: 60 })
+    .$defaultFn(() => Bun.randomUUIDv7())
+    .primaryKey()
+    .notNull(),
+  payerId: varchar({ length: 60 })
```

Comment:
How about we use senderId instead of payerId?
It would be natural senderId, receiverId, let me know your thoughts.

Reply:
Cool. `senderId` seems more reasonalble. Updated.